setup.cfg

```diff
@@ -16,7 +16,8 @@ setup_requires = setuptools_scm
 install_requires =
     sunpy>=3
     pyvista
-
+    matplotlib
+    scipy
```

Comment:
`tox -e build_docs`
Seems to always give me an error for mpl and scipy not being found because of the maps loaded in examples.

Reply:
Isn't that since they are not in the docs part of the requirements below?

Reply:
Even after adding `all` to `extras` I was experiencing, I'm not sure what the issue is

Reply:
All where?

Reply:
Nevermind, RTD still wasn't displaying the image so I'll revert back to the original code.